account_banking_pain_base/migrations/11.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    """Update database from previous versions, after updating module."""
+
+    if not version:
+        return
+    cr = env.cr
+
+    # update charge_bearer of account_payment_lines
+    plines = env['account.payment.line'].search([])
+    for pline in plines:
+        pline.compute_charge_bearer()
```

Comment:
This should not be necessary, since it is a new field, Odoo will compute it automatically.

Reply:
In fact the field is not a computed field: it has 'SLEV' by default which may not be right for every lines.